content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,16 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug:  dropbox-coffee-chats
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+## Event Details ##
+
++ **Who:** This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
also two more things for this line

can you have it say  "Who: Cis & trans women and non-binary individuals" instead? we started doing that recently, cause it seems to make more sense to have a noun as the answer to 'who' and also it makes it more clear and inclusive

and can you not have the period at the end (since no other line does)?

thanks <3